build/infrastructure/main/func-functionhost.tf

```
@@ -37,15 +37,15 @@ module "func_functionhost" {
     DOMAINEVENT_LISTENER_CONNECTION_STRING                          = module.sb_charges.primary_connection_strings["listen"]
     CHARGE_CREATED_TOPIC_NAME                                       = data.azurerm_key_vault_secret.sbt_charge_created_name.value
     CHARGE_PRICES_UPDATED_TOPIC_NAME                                = data.azurerm_key_vault_secret.sbt_charge_prices_updated_name.value
-    CHARGE_LINK_ACCEPTED_TOPIC_NAME                                 = module.sbt_link_command_accepted.name
+    CHARGE_LINKS_ACCEPTED_TOPIC_NAME                                 = module.sbt_links_command_accepted.name
     CHARGE_LINKS_REJECTED_TOPIC_NAME                                 = module.sbt_links_command_rejected.name
-    CHARGELINKACCEPTED_SUB_REPLIER                                  = "chargelinkaccepted-sub-replier"
-    CHARGELINKACCEPTED_SUB_EVENTPUBLISHER                           = "chargelinkaccepted-sub-eventpublisher"
-    CHARGELINKACCEPTED_SUB_DATAAVAILABLENOTIFIER                    = "chargelinkaccepted-sub-dataavailablenotifier"
-    CHARGELINKACCEPTED_SUB_CONFIRMATIONNOTIFIER                     = "chargelinkaccepted-sub-confirmationnotifier"
-    CHARGE_LINK_CREATED_TOPIC_NAME                                  = data.azurerm_key_vault_secret.sbt_charge_link_created_name.value
-    CHARGE_LINK_RECEIVED_TOPIC_NAME                                 = module.sbt_link_command_received.name
-    CHARGE_LINK_RECEIVED_SUBSCRIPTION_NAME                          = "link-command-received-receiver"
+    CHARGELINKSACCEPTED_SUB_REPLIER                                  = "chargelinksaccepted-sub-replier"
+    CHARGELINKSACCEPTED_SUB_EVENTPUBLISHER                           = "chargelinksaccepted-sub-eventpublisher"
+    CHARGELINKSACCEPTED_SUB_DATAAVAILABLENOTIFIER                    = "chargelinksaccepted-sub-dataavailablenotifier"
+    CHARGELINKSACCEPTED_SUB_CONFIRMATIONNOTIFIER                     = "chargelinksaccepted-sub-confirmationnotifier"
+    CHARGE_LINKS_CREATED_TOPIC_NAME                                  = data.azurerm_key_vault_secret.sbt_charge_link_created_name.value
+    CHARGE_LINKS_RECEIVED_TOPIC_NAME                                 = module.sbt_links_command_received.name
+    CHARGE_LINKS_RECEIVED_SUBSCRIPTION_NAME                          = "links-command-received-receiver"
```

As you're changing this anyway. Couldn't we also align the naming?

```suggestion
    CHARGE_LINKS_ACCEPTED_SUB_REPLIER                                  = "chargelinksaccepted-sub-replier"
    CHARGE_LINKS_ACCEPTED_SUB_EVENTPUBLISHER                           = "chargelinksaccepted-sub-eventpublisher"
    CHARGE_LINKS_ACCEPTED_SUB_DATAAVAILABLENOTIFIER                    = "chargelinksaccepted-sub-dataavailablenotifier"
    CHARGE_LINKS_ACCEPTED_SUB_CONFIRMATIONNOTIFIER                     = "chargelinksaccepted-sub-confirmationnotifier"
    CHARGE_LINKS_CREATED_TOPIC_NAME                                  = data.azurerm_key_vault_secret.sbt_charge_link_created_name.value
    CHARGE_LINKS_RECEIVED_TOPIC_NAME                                 = module.sbt_links_command_received.name
    CHARGE_LINKS_RECEIVED_SUBSCRIPTION_NAME                          = "links-command-received-receiver"
```

---

Will do